README.md

```diff
@@ -71,8 +71,6 @@ End-to-end tests for the Open Cluster Management search component. The tests con
 | SKIP_UI_TEST                 | Optional          | Option to skip the UI/Cypress E2E test                                                                                              |
 | TEST_ENV                     | Optional          | Test environment to run the E2E test  (**Options**: `canary`, `rosa`)                                                               |
 | TEST_MODE                    | Optional          | Test mode to run the E2E test  (**Options**: `BVT`, `smoke`) (**Alt**: `CYPRESS_TEST_MODE`)                                         |
-| USE_HUB_KUBECONFIG           | Optional          | Option to use `hub-kubeconfig` in E2E test (**Alt**: `CYRESS_USE_HUB_KUBECONFIG`)                                                   |
-| USE_MANAGED_KUBECONFIG       | Optional          | Option to use `import-kubeconfig` in E2E test (**Alt**: `CYRESS_USE_MANAGED_KUBECONFIG`)                                            |
```

Comment:
Removing `USE_HUB_KUBECONFIG`, for the hub cluster, we are only using the actual credentials that are passed within the environment or from the options.yaml file. For the managed cluster, instead of using either kubeconfig or cluster credentials, we'll focus on the kubeconfig usage, since that is what CICD prioritizes. Users can still export the creds for the managed cluster; however, we will only use the kubeconfig file that is either set or generated throughout the test run.